released/packages/coq-ltac-iter/coq-ltac-iter.1.0.0/opam

```diff
@@ -0,0 +1,20 @@
+opam-version: "2.0"
+maintainer: "gmalecha@gmail.com"
+homepage: "https://github.com/gmalecha/coq-ltac-iter"
+bug-reports: "https://github.com/gmalecha/coq-ltac-iter/issues"
+license: "MIT"
+build: [
+  [make "-j%{jobs}%"]
+  [make "install"]
```

Comment:
@clarus this package has `make "install"` in the `build` directive (and not `install`), which normally will prevent installation (due to `bwrap`). Can you please check locally that anything is actually installed? I'm worried this package is actually a no-op.